src/services/account.js

```diff
@@ -0,0 +1,18 @@
+const fetchAccount = async (id, setAccount, setLoading, setError) => {
+    const VITE_API_BASE_URL = import.meta.env.VITE_API_BASE_URL;
+    try {
+      const response = await fetch(`${VITE_API_BASE_URL}/accounts/${id}`);
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      const data = await response.json();
+      setAccount(data);
+      setLoading(false);
```

Comment:
Keep the state mechanisms like setAccount and setAccount as part of the component. The service needs to be state less

Reply:
Great point!